drivers/debian/etc/modprobe.d/tier4-isx021.conf

```diff
@@ -5,10 +5,19 @@
 # "enable_auto_exposure=0" disable auto-exposure feature. 
 # "enable_auto_exposure=1" enable auto-exposure feature. (Default)
 #
-# "enable_distortion_correction=0" disable distortion correction feature. (Default)
-# "enable_distortion_correction=1" enable distortion correction feature.
+# "enable_distortion_correction=0" disable distortion correction feature.
+# "enable_distortion_correction=1" enable distortion correction feature. (Default)
+#
+# "fsync_mode=0" disable FPGA Fsync mode. (Default)
```

Comment:
Could you add a note that it is only valid for RQX-58G/RQX-59G?

Reply:
Sure, please check this commit: https://github.com/tier4/tier4_automotive_hdr_camera/commit/41dccf226e9557654336c70ab39dd0311a75dc68
Thank you!